### PR TITLE
feat: playlist page and refactoring

### PR DESCRIPTION
**Pull request by pankaj-prog**
- added routes with navbar( where the sidebar is not shown ) 
- feat: playlist page: 

BlitheBrains - playlist

### CODE REVIEW

File: src/pages/Playlist/Playlist.jsx [hunk 1]
```diff
@@ -1,8 +1,64 @@
...
+      <p className="text-center">
+        <button className="btn btn-solid-primary btn-rc gutter-bottom-24">
+          Create new playlist
+        </button>
+      </p>
+      <div className="playlist-page">
+        <section className="playlist-wrapper">
+          {playlists.map((playlist) => {
+            return (
+              <div className="text-center">
```
Comment (gauravsinhaweb) [inline src/pages/Playlist/Playlist.jsx]: key is need to be passed here

File: src/pages/Playlist/Playlist.jsx [hunk 2]
```diff
@@ -1,8 +1,64 @@
...
+                {playlist._id == "a2" && (
+                  <p className="playlist-description">{playlist.description}</p>
+                )}
+              </div>
+            );
+          })}
+        </section>
+        <section className="playlist-videos">
+          {videos.map((video) => (
+            <VideoCardHorizontal {...video} cardType="playlist" />
```
Comment (gauravsinhaweb) [inline src/pages/Playlist/Playlist.jsx]: key??

**Review by gauravsinhaweb — APPROVED** [review]
some improvements need to be done, I mentioned, overall everything looks good! Good to merge after changes!

File: src/pages/Playlist/Playlist.jsx [hunk 3]
```diff
@@ -1,8 +1,68 @@
...
+    <div className=" content">
+      <Banner title="Playlists" />
+      <p className="text-center">
+        <button className="btn btn-solid-primary btn-rc gutter-bottom-24">
+          Create new playlist
+        </button>
+      </p>
+      <div className="playlist-page">
+        <section className="playlist-wrapper">
+          {playlists.map((playlist) => {
```
Comment (bhtibrewal) [inline src/pages/Playlist/Playlist.jsx:36]: Suggestion: you should destructure the playlist here

File: src/pages/Playlist/Playlist.css [hunk 4]
```diff
@@ -0,0 +1,35 @@
+.playlist-page {
+  display: grid;
+  grid-template-columns: 25% 1fr;
+}
+
+.playlist-wrapper {
+  margin: 0 2rem 0 0;
```
Comment (bhtibrewal) [inline src/pages/Playlist/Playlist.css:7]: Suggestion: write margin-top here

**Review by bhtibrewal — APPROVED** [review]
The UI and code both look good, you are good to merge

**Review by m26unkwn — APPROVED** [review]
could not find any issue you can check the comments given by the others.
otherwise good to go

**Pull request merged**